pkg/trace/testutil/trace.go

```diff
@@ -89,9 +89,7 @@ func RandomTrace(maxLevels, maxSpans int) pb.Trace {
 
 // RandomTraceChunk generates a random trace chunk with a depth from 1 to
 // maxLevels of spans. Each level has at most maxSpans items.
-//
-//nolint:revive // TODO(APM) Fix revive linter
-func RandomTraceChunk(maxLevels, maxSpans int) *pb.TraceChunk {
+func RandomTraceChunk(maxLevels, _ int) *pb.TraceChunk {
```

Comment:
huh, this looks like a bug? The description and the usage here don't match - maybe we can just fix this instead by passing maxSpans into the `RandomTrace` call below? (Or do you want to keep this PR just fixing the lint comments in which case this is fine)

Reply:
Yes nice catch! I'll do it in a separate PR in case I'll need to change the tests relying on the util.